src/components/PercentBar.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
The style of this file is a little different from the rest of the files. It has single quotes and slightly different indentation. No big deal of course! Who case about ' vs "? Consistency is nice though.

There's a `.prettierrc.json` file in the repo root. If you run prettier on your contributions, they should all just end up formatted the same as everything else.

Reply:
Handled in 59aa6e5.